development/build/scripts.js

```diff
@@ -920,6 +969,24 @@ function setupMinification(buildConfiguration) {
   });
 }
 
+function setupScuttlingWrapping(buildConfiguration, applyLavaMoat) {
```

Comment:
in stead of appending scuttling can we just add a script file that scuttles?

Reply:
scuttling takes place in LavaMoat, but in order for sentry to still work under scuttled environment I have to wrap its entire code with `with`s and `proxy`s, that generate a safe-from-scuttled-env scope - so in that sense it kinda has to be textually wrapping sentry source code in one file rather than loading it separately.